apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -355,6 +475,32 @@ public RifFile toRifFile() {
     };
   }
 
+  /**
+   * @param dataSetLocation the {@link TestDataSetLocation} of the file to get a local copy of
+   * @param fileName the name of the specific file in the specified {@link TestDataSetLocation} to
+   *     get a local copy of, e.g. "beneficiaries.rif"
+   * @return a {@link URL} to a local copy of the specified test data file from S3
+   */
+  private static Supplier<URL> localData(TestDataSetLocation dataSetLocation, String fileName) {
+    return () -> {
+      // Find the build output `target` directory.
+      Path resourceDir = Paths.get(System.getenv("LOCAL_SYNTHETIC_DATA"));
```

Comment:
Same as with the last PR review, could you document that parameter in a `README` somewhere?